### DISCUSSION TITLE
Could not find a declaration file for module 'react-query'

### DISCUSSION THREAD

**Discussion by yajun-liu** [TanStack/query#3872] [Q&A]
I am testing https://github.com/TkDodo/testing-react-query.git from TkDodo and failed to compile:

Could not find a declaration file for module 'react-query'
Try `npm i --save-dev @types/react-query` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-query';`

**Comment by TkDodo** [comment]
I need to update it to import from TanStack/query.

**Reply by yajun-liu** [reply]
It works when import from @tanstack/react-query.

Thanks.
